Formula/tuntox.rb

```diff
@@ -0,0 +1,49 @@
+class Tuntox < Formula
+  desc "Tunnel TCP connections over the Tox protocol"
+  homepage "https://gdr.name/tuntox/"
+  url "https://github.com/gjedeer/tuntox/archive/refs/tags/0.0.10.1.tar.gz"
+  sha256 "7f04ccf7789467ff5308ababbf24d44c300ca54f4035137f35f8e6cb2d779b12"
+  license "GPL-3.0-only"
+  head "https://github.com/gjedeer/tuntox.git", branch: "master"
+
+  depends_on "cscope" => :build
+  depends_on "pkg-config" => :build
+  depends_on "toxcore"
+
+  def install
+    inreplace "gitversion.h", /.*/, "#define GITVERSION \"4eda1442c458506158257341ab6d3fd4543288cc\""
+
+    if OS.mac?
+      inreplace "Makefile.mac", ".git/HEAD .git/index", ""
+      system "make", "-f", "Makefile.mac", "LIB_DIR=#{HOMEBREW_PREFIX}/lib"
+      bin.install "tuntox"
+    else
+      inreplace "Makefile" do |s|
+        s.gsub! "%.o: %.c $(INCLUDES) gitversion.h", "%.o: %.c $(INCLUDES)"
+        s.gsub! "gitversion.h: FORCE", "gitversion.h:"
+      end
+      system "make", "tuntox_nostatic"
+      bin.install "tuntox_nostatic" => "tuntox"
+    end
```

Comment:
```suggestion
    inreplace "gitversion.h", /.*/, '#define GITVERSION "N/A"'
    system "make", "prefix=#{prefix}", "install"
```
Let's try this.

Reply:
I think we might still need to keep some of the `inreplace`s for the dependencies on the local git repo. It seems like maybe the `FORCE` target makes the git portion run.

Do you think this is fine?

`inreplace "gitversion.h: FORCE", "gitversion.h:"`